Client/src/Remote/Endpoints/UserEndpoint.tsx

```diff
@@ -16,4 +20,10 @@ export const updateUserDetails = async (userProfile: IUserDetailsModel): Promise
 export const getUserDetails = async (userDetailsQuery: IUserDetailsQuery): Promise<IUserAccountModel> => {
   const remoteUser = await get(userDetailsRoute).withQuery(userDetailsQuery).json()
   return toLocalUserAccountModel(remoteUser)
+}
+
+export const getUserFlaggedDatasets = async (): Promise<IApprovedDatasetModel[]> => {
+  const remoteDatasets = await get(userFlaggedDatasetsRoute).json()
+  const localDatasets = toLocalApprovedDatasets(remoteDatasets)
```

Comment:
I hope the backend returns an IApprovedDatasetModel[] right away wihtout the need to map this, map is very error prone, backend should provide a clean model right away

Reply:
It doesnt. The mapping is required